erpnext/stock/doctype/batch/batch.js

```diff
@@ -22,8 +22,18 @@ frappe.ui.form.on('Batch', {
 				frappe.set_route("query-report", "Stock Ledger");
 			});
 			frm.trigger('make_dashboard');
+			frm.add_custom_button(__('Make Material Request'), () => {
+				frm.trigger("make_request")
+			});
```

Comment:
Put this under a "Create" dropdown, and add the primary class to the Create button:

```suggestion
			frm.add_custom_button(__('Material Request'), () => {
				frm.trigger("make_request")
			}, __("Create"));

			this.frm.page.set_inner_btn_group_as_primary(__('Create'));
```